android/modules/platform/src/java/ti/modules/titanium/platform/PlatformModule.java

```diff
@@ -7,34 +7,44 @@
 package ti.modules.titanium.platform;
 
 import org.appcelerator.kroll.KrollDict;
+import org.appcelerator.kroll.KrollInvocation;
 import org.appcelerator.kroll.KrollModule;
 import org.appcelerator.kroll.KrollProxy;
 import org.appcelerator.kroll.KrollRuntime;
 import org.appcelerator.kroll.annotations.Kroll;
 import org.appcelerator.kroll.common.Log;
+import org.appcelerator.titanium.io.TiBaseFile;
+import org.appcelerator.titanium.io.TiFileFactory;
+import org.appcelerator.titanium.io.TiFileProvider;
 import org.appcelerator.titanium.TiApplication;
 import org.appcelerator.titanium.TiC;
+import org.appcelerator.titanium.util.TiMimeTypeHelper;
 import org.appcelerator.titanium.util.TiPlatformHelper;
+import org.appcelerator.titanium.util.TiUrl;
 
 import android.app.Activity;
 import android.content.ActivityNotFoundException;
```

Comment:
Not related to this PR, but we can clean up the `import android.content.ActivityNotFoundException;` import.

Reply:
Thanks. I'll remove it.